variables.tf

```diff
@@ -146,6 +146,57 @@ variable "attach_policies" {
   default     = false
 }
 
+
+##########################
```

Comment:
Move all cloudwatch vars to a better place (eg, line 57). Now they are in the middle of the policies section.